### PR TITLE
added new urls to theme

### PR DESCRIPTION
**Pull request by amirtds**

### CODE REVIEW

**Comment by amirtds** [comment]
[(4) Trinity: replace Sign in / Register button links with OAuth links](https://trello.com/c/H2EuXOrv/1866-4-trinity-replace-sign-in-register-button-links-with-oauth-links)

File: lms/templates/design-templates/header/_trinity-header.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+             <ul>
+               <li>
+                 <a href="/courses">${_("Find Courses")}</a>
+               </li>
+             </ul>
+           </li>
+           <li class="a--site-menu__group">
+             <ul>
+               <li class="a--nav-cta">
+                 <a href="${custom_login_url}">Trinity Account</a>
```
Comment (melvinsoft) [inline lms/templates/design-templates/header/_trinity-header.html]: I think for now the button text should just login, they haven't requested any special text, and they don't use trinity as name I think. But is a good question to ask them.

File: lms/templates/design-templates/header/_trinity-header.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+                   </li>
+                 % endfor
+                 <li class="a--spacer"></li>
+               % endif
+               <li>
+                 <a href="/courses">${_("Find Courses")}</a>
+               </li>
+               <li class="a--spacer"></li>
+               <li>
+                 <a href="${custom_login_url}">Trinity Account</a>
```
Comment (melvinsoft) [inline lms/templates/design-templates/header/_trinity-header.html]: @amirtds Same as above.

**Comment by melvinsoft** [comment]
@amirtds Great work, this is looking good, I just requested a couple of minor changes.

**Pull request merged**